packages/quantic/force-app/main/default/lwc/quanticCaseClassification/quanticCaseClassification.html

```diff
@@ -0,0 +1,65 @@
+<template>
+  <fieldset class={formClass}>
+    <legend class="slds-form-element__legend slds-form-element__label slds-text-heading_small">
+      {label}
+    </legend>
+    <div class="slds-form-element__control" aria-describedby="case-classification-error-message">
+      <div class="row">
+        <template if:true={loading}>
+          <div class="loading-holder slds-var-m-top_x-small">
+            <lightning-spinner alternative-text="Loading" size="small"></lightning-spinner>
+          </div>
+        </template>
+        <template if:false={loading}>
+          <template for:each={suggestions} for:item="option">
```

Comment:
```suggestion
          <template for:each={suggestions} for:item="suggestion">
```
Just to be consistent? Do you prefer that name?